Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -799,7 +799,7 @@ static NetworkIdentity SpawnSceneObject(SpawnMessage msg)
                 if (logger.LogEnabled())
                 {
                     foreach (KeyValuePair<ulong, NetworkIdentity> kvp in spawnableObjects)
-                        logger.Log("Spawnable: SceneId=" + kvp.Key + " name=" + kvp.Value.name);
+                        if (logger.LogEnabled()) logger.Log("Spawnable: SceneId=" + kvp.Key + " name=" + kvp.Value.name);
```

Comment:
```suggestion
                        logger.Log("Spawnable: SceneId=" + kvp.Key + " name=" + kvp.Value.name);
```
check is on line above

Reply:
fixed